composer.json

```diff
@@ -18,7 +18,7 @@
         "composer/semver": "^3.4",
         "composer/xdebug-handler": "^3.0.5",
         "doctrine/inflector": "^2.0.10",
-        "illuminate/container": "^11.22.0",
+        "illuminate/container": "11.22.3",
```

Comment:
There is no 11.22.3 version, probably you mean 11.23.5, see https://packagist.org/packages/illuminate/container

Reply:
```suggestion
        "illuminate/container": "11.23.5",
```